test/compile-test.cc

```diff
@@ -23,6 +23,7 @@
 #endif
 
 #include "fmt/compile.h"
+#include "fmt/ostream.h"
```

Comment:
compile-test shouldn't depend on ostream. I think the test in ostream-test is enough.

Reply:
commit with this changes reverted